src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -353,6 +386,35 @@ private IEnumerable<StandardVersionApplication> ApplyVersionStatuses(IEnumerable
                 }
             }
 
+            var withdrawals = previousWithdrawals.Where(x => x.StandardApplicationType == StandardApplicationTypes.VersionWithdrawal 
```

Comment:
This isn't what I meant when checking for Approved in the previous comment, You want approved withdrawal applications don't you?

Reply:
Yes - this has been corrected